templates/submission/submissionMetadataFormFields.tpl

```diff
@@ -9,6 +9,11 @@
  * submission metadata.
  *}
 
+{if $citationsEnabled && array_intersect(array(ROLE_ID_REVIEWER, ROLE_ID_AUTHOR), (array)$userRoles)}
```

Comment:
@bozana, this has the side-effect of causing the citation form to not display for e.g. editors who are submitting but don't have an author role. (https://forum.pkp.sfu.ca/t/field-references-doesnt-show-on-the-submission-form-even-if-enabled-ojs3/43440/1)

Reply:
Thanks @asmecher!
I'll also note for myself the other forum entry here: https://forum.pkp.sfu.ca/t/ojs-3-1-1-0-article-statistics-reference-box/40633/15
Will change...